packages/flutter/lib/src/material/theme_data.dart

```diff
@@ -789,6 +799,7 @@ class ThemeData extends Diagnosticable {
     CupertinoThemeData cupertinoOverrideTheme,
     SnackBarThemeData snackBarTheme,
     BottomSheetThemeData bottomSheetTheme,
+    PopupMenuTheme popupMenuTheme,
```

Comment:
This should still be `PopupMenuThemeData`

Reply:
Done